sdk/deviceupdate/Azure.ResourceManager.DeviceUpdate/api/Azure.ResourceManager.DeviceUpdate.netstandard2.0.cs

```diff
@@ -442,6 +443,12 @@ public partial class DiagnosticStorageProperties
         public string ConnectionString { get { throw null; } set { } }
         public string ResourceId { get { throw null; } set { } }
     }
+    public partial class Encryption
```

Comment:
this model name is too generic. Could we be more specific? For instance, we could prepend your library name `DeviceUpdate` on this so that we have `DeviceUpdateEncryption`

Reply:
@ArcturusZhang We are fine to name it DeviceUpdateEncryption in the SDK but we would like to keep "Encryption" in the swagger given that's how other services use it.

Reply:
Made the property rename change in SDK , swagger will still have Encryption property.